include/silo/database.h

```diff
@@ -36,6 +44,12 @@ class Database {
    std::unique_ptr<ReferenceGenome> reference_genome;
    std::vector<DatabasePartition> partitions;
 
+   std::unordered_map<std::string, storage::column::StringColumn> string_columns;
+   std::unordered_map<std::string, storage::column::IndexedStringColumn> indexed_string_columns;
+   std::unordered_map<std::string, storage::column::IntColumn> int_columns;
+   std::unordered_map<std::string, storage::column::DateColumn> date_columns;
+   std::unordered_map<std::string, storage::column::PangoLineageColumn> pango_lineage_columns;
```

Comment:
Sorry, another naming discussion: :)

Why are those "columns" called column? 
* The int column does nothing at all. -> Why do we need this at all?
* The date column stores `is_sorted` which is redundant. -> Why do we need this at all?
* The (indexed) string and pango lineage columns serve as lookup -> Bundle them in a class `ValueLookup` or `Dictionary` and rename them to something like `StringValueLookup`?

Reply:
Btw I expect the same for float columns. Probably we don't need the global float column, do we?

Reply:
I see your points. The fact that they serve as factories for column partitions encapsulated the internal logic of columns and partitions away from the user which was quite nice in my opinion.

Reply:
Ah, I didn't see that. That's a fair point to keep them then.